### PR TITLE
Unskip and fix test CS0030ERR_NoExplicitConv02

### PR DESCRIPTION
**Pull request by VladimirReshetnikov**
Unskip test CS0030ERR_NoExplicitConv02 and remove its dependency on the current culture.
Further error message improvement is tracked as issue #920.

### CODE REVIEW

**Comment by VladimirReshetnikov** [comment]
@AlekseyTs @tmat @VSadov Please review.

File: src/Compilers/CSharp/Test/Semantic/Semantics/SemanticErrorTests.cs [hunk 1]
```diff
@@ -1476,10 +1478,19 @@ public class iii
...
+        [WorkItem(920, "http://github.com/dotnet/roslyn/issues/920")]
+        [Fact]
         public void CS0030ERR_NoExplicitConv02()
         {
-            var text = @"
+            var savedCurrentCulture = Thread.CurrentThread.CurrentCulture;
+            var savedCurrentUICulture = Thread.CurrentThread.CurrentUICulture;
+
+            Thread.CurrentThread.CurrentCulture = CultureInfo.InvariantCulture;
+            Thread.CurrentThread.CurrentUICulture = CultureInfo.InvariantCulture;
```
Comment (AlekseyTs) [inline src/Compilers/CSharp/Test/Semantic/Semantics/SemanticErrorTests.cs]: Let's change the culture only for the duration of VerifyDiagnostics method, but we should request diagnostics before the culture is changed

Comment (VladimirReshetnikov) [inline src/Compilers/CSharp/Test/Semantic/Semantics/SemanticErrorTests.cs]: @AlekseyTs Done.

**Comment by AlekseyTs** [comment]
LGTM

**Comment by VSadov** [comment]
LGTM

**Pull request merged**